front/app/components/FormBuilder/components/FormBuilderSettings/PointSettings/index.tsx

```diff
@@ -70,35 +70,30 @@ const PointSettings = ({ mapConfigIdName, field }: Props) => {
   }, [localize, mapConfig]);
 
   const onConfigureMapClick = useCallback(() => {
-    // Create a new map config if we don't have one for this field
+    // Create a new map config by duplicating the project config if we don't have one for this field
     if (!mapConfigId) {
-      // Initial data is from existing project map config
-      const initialData = projectMapConfig?.data?.attributes
-        ? projectMapConfig.data.attributes
-        : {};
-
-      createProjectMapConfig(
-        {
-          ...initialData,
-        },
-        {
+      const projectMapConfigId = projectMapConfig?.data?.id;
+
+      // TODO: What should happen if we don't have a project map config? Should this ever happen?
```

Comment:
@jamesspeake If we don't have a project map config, we need to create a new one for this custom field (using `createProjectMapConfig`) :)

I'm not sure if we create a new map config if the BE will by default populate it with the global map default values (the ones set in AdminHQ)? :thinking:

Reply:
Is that possible that we don't have one?

Reply:
@jamesspeake Yeah - it's one thing I really don't like about project-level maps today :sweat_smile: They aren't created when a project is, they are only created once an Admin visits the "Map" tab in the back office for a project.

So if the project doesn't have an ideation phase, or if the admin hasn't visited the "Map" tab before in a project, there won't be a project map config.